sdk/communication/azure-communication-email/tests.yml

```diff
@@ -21,13 +21,17 @@ extends:
             - $(sub-config-azure-cloud-test-resources)
             - $(sub-config-communication-services-cloud-test-resources-common)
             - $(sub-config-communication-services-cloud-test-resources-java)
+          SubscriptionConfigurationFilePath: eng/common/TestResources/sub-config/AzurePublicMsft.json
+          ServiceConnection: azure-sdk-tests
         Int:
           SubscriptionConfigurations:
             - $(sub-config-communication-int-test-resources-common)
             - $(sub-config-communication-int-test-resources-java)
+          ServiceConnection: azure-sdk-tests
         PPE:
           SubscriptionConfigurations:
             - $(sub-config-communication-ppe-test-resources-common)
             - $(sub-config-communication-ppe-test-resources-java)
+          ServiceConnection: azure-sdk-tests
```

Comment:
The `ServiceConnection` for INT and PPE env is pending on EngSys. This is not specific to Java but applicable cross language